packages/edit-site/src/components/global-styles/font-library-modal/font-collection.js

```diff
@@ -186,6 +203,24 @@ function FontCollection( { slug } ) {
 		resetFontsToInstall();
 	};
 
+	let footerComponent = null;
+	if ( selectedFont ) {
+		footerComponent = (
+			<InstallFooter
+				handleInstall={ handleInstall }
+				isDisabled={ fontsToInstall.length === 0 }
+			/>
+		);
+	} else if ( ! renderConfirmDialog ) {
```

Comment:
```suggestion
	} else if ( ! renderConfirmDialog && totalPages > 0 ) {
```

To prevent unintended pagination from being displayed if the font is not found.

![image](https://github.com/WordPress/gutenberg/assets/54422211/345c07b6-a0bf-4583-9bc6-183239110f41)

Reply:
Thank you. I've changed the value to 1 so that pagination is also hidden when there are no pages beyond the first to go to.